how_to_submit_a_pr_-_quick_guide.md

```diff
@@ -0,0 +1,83 @@
+# How to submit a Pull Request - Quick guide
+
+
+## Objective
+
+After reading, you will know how to submit Pull Request (PR) for code review.
+
+## Prerequisite
+
+* I Understand the project goal and what is its main purpose.
+* I know who is the team lead and account manager.
+
+
+
+## The Flow
+
+1. **Open your task** on Github: Go to `Project` tab, click on `Weekly Workplan`, under your name you will see all your tasks in the project.
+2. Read the description and move the issue to `Today in Progress`. **Check if you understand the issue**, and if you don't, this is the time to  write any questions. *This is important. Keep asking questions until you understand!*
+3. **Check the timeboxing** and see if it make sense to you. If it doesn’t, communicate that to your team lead. *This will help identify early on any issues with the timebox, how well the task was explained, and if you really understand the task. Giving this thought can save you a lot confusion and frustration later on.  *
+4. **Write user stories** (in [Behat format](http://docs.behat.org/en/v2.5/guides/1.gherkin.html)), to be used for the QA team and automated test.
```

Comment:
Is there any further explanation for "to be used for the QA team and automated test."  This is probably enough but just wanted to check.